docs/troubleshooting.md

```diff
@@ -0,0 +1,81 @@
+  for the full list.
+* Ensure you have enough [quota](https://cloud.google.com/compute/quotas) in the
+  zone/region running the pipeline. By default, the pipeline runs in the
+  `us-central1` region. You may change this by specifying `--region <region>`
+  or `--zone <zone>` when running the pipeline. You can check for quota issues
+  by navigating to the
+  [Compute Engine quotas page](https://console.cloud.google.com/iam-admin/quotas?service=compute.googleapis.com)
+  while the pipeline is running, which shows saturated quotas at the top of the
+  page.
+* `gzip` and `bzip2` file formats cannot be sharded, which considerably slows
+  down the pipeline. Consider decompressing the files prior to running the
+  pipeline. You may use [dsub](https://github.com/googlegenomics/dsub) to write
+  a script to decompress the files in scalable manner. Note that this is only an
+  issue if running the pipeline with a small number of large files (i.e. running
+  with a large number of small files is usually fine as each file can be read by
+  a separate process).
+
+See [handling large inputs](./large_inputs.md) for more details.
+
+## Pipeline crashes due to out of disk error
+
+Try increasing the disk size allocated to each worker by specifying
+`--disk_size_gb <disk_size>` and/or increasing the number of workers by
+specifying `--max_num_workers <num_workers>`. By default, each worker gets 250GB
+of disk, which may not be enough to read the entire dataset.
+
+See [handling large inputs](./large_inputs.md) for more details.
+
+## Error: "JSON parsing error ... No such field: <field_name>"
+
+If you see an error in the form,
+`Error while reading data, error message: JSON parsing error in row starting at
+position 0: No such field: <field_name>`, it means that the `<field_name>`
+is missing from the BigQuery schema, which is caused by its definition missing
+from a VCF header file.
+
+You can fix this by:
+
+* Changing the VCF file containing the field and adding an entry in the header
+  with proper definition. For instance, if the error is for the field `AF`, you
+  can add the following to the VCF file:
+
+  ```
+  ##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
+  ```
+
+  Note that it's important to provide valid `Type` and `Number` for the field.
+  If unsure, you can provide `Type=String` and `Number=.` (i.e. a generic list
+  of strings), which will match any field. Please also check whether the
+  missing field is for `##INFO` or `##FORMAT`.
+
+* If changing the file(s) is not an option, you can also run the pipeline with
+  `--representative_header_file <file_path>`, where you provide a merged view
+  of all headers in all files. You can add any missing fields to that file.
+  We are working on a tool to make this process easier and provide
+  recommandations for missing fields.
+
+* Coming soon: Run the pipeline with `--infer_undefined_headers`. This will do
```

Comment:
Why coming soon? This has been added since long time ago. :-)

Reply:
Right, but the docker image hasn't been updated and it's not yet tested well (there are no integration tests for it). Also, I thought that it wouldn't work without your recent PR to covert list and single-value numbers?

Reply:
Docker image contains this flag (built on Feb 27, and PR#114 submitted on Feb 23).
Infer header logic works, but it wont be enough if the value provided for the field is empty (and no = sign is present) because PyVcf extracts the data as Flag which cannot be casted to string(this case is handled in PR#146).

Reply:
yeap, so I think it's best if we wait for the new release and update this doc once we have added some integration testing.